test/e2e/suite_test.go

```diff
@@ -109,7 +104,7 @@ var _ = ginkgo.BeforeSuite(func() {
 
 	err = setupTestNamespace(testNamespace, kubeClient)
 	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
-}, TestSuiteSetupTimeOut.Seconds())
+})
```

Comment:
What's the timeout now?

Reply:
Maybe it's the reason: https://onsi.github.io/ginkgo/MIGRATING_TO_V2#timeout-behavior